### ISSUE TITLE
Deploying to Posit Cloud is no longer in a closed beta

### ISSUE THREAD

**Issue by jennybc** [rstudio/rsconnect#855]
This part of the docs states that deploying to Posit Cloud is in a closed beta:

https://github.com/rstudio/rsconnect/blob/c31971db3b97024420413536538be3206f57a9b4/index.html#L99

> Publishing to Posit Cloud is currently in closed beta and requires access to an enabled account on Posit Cloud. Premium, Instructor, and Organization account holders can request to participate. See [Posit Cloud’s Announcement](https://posit.cloud/learn/whats-new#publishing) for more information and to request access.

The README appears to be current and reflects that this feature is in GA. Not sure why the website doesn't reflect that.

**Comment by aronatkins** [comment]
Yeah. The shift from beta happened after our last CRAN release and so much has changed since then that cutting a new release isn't lightweight.

@hadley - is it worth a PR to scrub this from the gh-pages?

https://github.com/rstudio/rsconnect/blob/c31971db3b97024420413536538be3206f57a9b4/index.html#L99

**Comment by jennybc** [comment]
If a release isn't imminent, this could be a time to do the "update a released site" thing:

https://pkgdown.r-lib.org/articles/how-to-update-released-site.html

**Comment by aronatkins** [comment]
Fixed with https://github.com/rstudio/rsconnect/commit/e6f1455134376f47f8529e412362644d5f91ccf8
